### PR TITLE
[ADI] Send contribution notification mail to idea author every 4 days after publication of an idea

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

File: src/Entity/IdeasWorkshop/Idea.php [hunk 1]
```diff
@@ -286,6 +286,13 @@ class Idea implements AuthorInterface, ReportableInterface, EnabledInterface
      */
     private $finalizedAt;
 
+    /**
+     * @var \DateTimeInterface
+     *
+     * @ORM\Column(type="date", nullable=true)
+     */
+    private $lastNotifiedAt;
```
Comment (ottaviano) [inline src/Entity/IdeasWorkshop/Idea.php]: what about: 
`lastContributionNotificationDate`
or
`contributionNotifiedAt`
?

Comment (illusionOfParadise) [reply src/Entity/IdeasWorkshop/Idea.php]: I like `lastContributionNotificationDate`.

File: src/Command/NotifyIdeaAuthorAboutContributionsCommand.php [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    {
+        if (1 > $interval = (int) $input->getOption('interval')) {
+            $this->io->error("Option 'interval' should be a valid integer different from zero.");
+
+            return 1;
+        }
+
+        foreach ($this->getIdeas($interval) as $idea) {
+            $this->sendMail($idea);
+            $idea->setlastContributionNotificationDate(new \DateTime());
```
Comment (ottaviano) [inline src/Command/NotifyIdeaAuthorAboutContributionsCommand.php]: `setLast...`

File: app/config/services/system.xml [hunk 3]
```diff
@@ -158,7 +158,7 @@
         </service>
 
         <!-- Dynamic redirections "to path" handler -->
-        <service id="app.react_routing_loader" class="AppBundle\React\ReactAppRegistry" />
+        <service id="app.react_routing_loader" class="AppBundle\React\ReactAppRegistry" public="true" />
```
Comment (ottaviano) [inline app/config/services/system.xml]: why this change?

Comment (illusionOfParadise) [reply app/config/services/system.xml]: Reverted.
